include/odp/api/spec/event.h

```diff
@@ -79,6 +79,20 @@ odp_event_type_t odp_event_types(odp_event_t event,
 int odp_event_type_multi(const odp_event_t event[], int num,
 			 odp_event_type_t *type);
 
+/**
+ * Event user area
+ *
+ * Returns pointer to the user area associated with the event. This maps to the
+ * user area of underlying event type (e.g. odp_packet_user_area() for packets).
+ * If the event does not have user area, NULL is returned.
+ *
+ * @param      event    Event handle
+ *
+ * @return Pointer to the user area of the event
+ * @retval NULL  The event does not have user area
+ */
+void *odp_event_user_area(odp_event_t event);
```

Comment:
Any sense providing a user area size API as well?

Reply:
We discussed about this during the last sync call and decided to not add such API at this point. The user area size function is currently available only for packets anyway.